src/installer/getScript.ts

```diff
@@ -21,6 +21,11 @@ ${huskyIdentifier}
 scriptPath="${script}.js"
 hookName=\`basename "$0"\`
 gitParams="$*"
+
+if [[ ! -z "$\{HUSKY_DEBUG\}" ]]; then
```

Comment:
This will only check if the environment variable does not have a zero length, not if it's set to `"true"`.

```sh
# all of these will result in printing a debug message
HUSKY_DEBUG=1
HUSKY_DEBUG=0
HUSKY_DEBUG=asdf1234teralyanything
HUSKY_DEBUG=true
HUSKY_DEBUG=false # this will still show a debug message!
```

The current implementation is somewhat similar to this:

https://github.com/typicode/husky/blob/b7d4a69f6d0e852b046e1ea21501c333cecf7630/src/installer/bin.ts#L6-L8

Is there a need to check if the variable is explicitly set to `true`?

Reply:
I think it's fine like this 👍